### ISSUE TITLE
Categorized symbology

### ISSUE THREAD

**Issue by honza2** [jirik/layman#303]
After uploading input layer to server and downloading back layer symbology does not respect original categorized distribution. I found difference in renderer-v2 element in QMLs. Thumbnail seems allright.
[category_test.zip](https://github.com/jirik/layman/files/6228091/category_test.zip)

**Comment by index-git** [comment]
I am not able to reproduce it. I load input.geojson with input.qml into Layman (v 1.11) and call GET Workspace Layer Style and get qml with categories element and without rules element. There are still differences like lowercase of attribute names.
[michal_output.zip](https://github.com/jirik/layman/files/6262829/michal_output.zip)

**Comment by jirik** [comment]
Closing, identified bug in plugin, not in Layman
